addon/helpers/flatten.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+const {
+  Helper: { helper }
+} = Ember;
+
+export function flatten(array) {
+  array = array || [];
+
+  return array.reduce((flat, toFlatten) => {
```

Comment:
This helper will not recompute when an array has changes. Please see something like the `contains` helper for an example.